src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
+  if (operation === '%') {
```

Comment:
- [ ]  [OPTIONAL] In fact, this operator should return percentage, not modulo. 
![1](https://user-images.githubusercontent.com/58552748/130351133-fd843b0d-05d0-40a7-a45d-ed8e32092d2f.png)